src/einsteinpy/symbolic/ricci.py

```diff
@@ -31,11 +32,12 @@ def __init__(self, arr, syms, config="ll", parent_metric=None):
             syms is not a list or tuple
         ValueError
             config has more or less than 2 indices
-        
+
         """
         super(RicciTensor, self).__init__(arr, config=config)
         self._order = 2
         self._parent_metric = parent_metric
+        self._invmetric = None
```

Comment:
No need for `_invmetric`, it's an attribute for `Metric` class only.

Reply:
Also can you add the attribute `parent_metric` in docstring of `__init__(self)` ?
Just take reference from `RiemannCuravtureTensor`